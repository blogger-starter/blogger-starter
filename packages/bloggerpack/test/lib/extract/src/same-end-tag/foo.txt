<example>
foo
</end>

<example2>
foo2
</end>

<example3>
foo3
</end>
