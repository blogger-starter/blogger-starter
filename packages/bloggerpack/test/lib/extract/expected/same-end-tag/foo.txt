/* src/same-end-tag/foo.txt */

foo
